UI/TechTreeWnd.cpp

```diff
@@ -1868,9 +1869,9 @@ void TechTreeWnd::TechListBox::Update(bool populate /* = true */) {
         auto& tech_row = row.second;
         if (TechVisible(tech_row->GetTech(), m_categories_shown, m_tech_statuses_shown)) {
             tech_row->Update();
-            insertion_timer.restart();
+            insertion_timer.start();
             auto listbox_row_it = Insert(tech_row);
-            insertion_elapsed += insertion_timer.elapsed();
+            insertion_elapsed += ElapsedSeconds(insertion_timer);
```

Comment:
probably need an optional parameter to control rounding to avoid issues when accumulating the results like this...